integ/src/updater.rs

```diff
@@ -72,6 +74,34 @@ pub async fn process_pods_test(action: Action, kube_config_path: &str) -> Update
     Ok(())
 }
 
+// =^..^=   =^..^=   =^..^=   =^..^=   =^..^= Deletion and Creation of cert manager  =^..^=   =^..^=   =^..^=   =^..^=   =^..^=
+
+// install or destroy cert-manager
+pub async fn process_cert_manager(action: Action, kube_config_path: &str) -> UpdaterResult<()> {
+    let action_string: String = action.to_string();
+
+    // install cert-manager
+    let cert_manager_status = Command::new(KUBECTL_BINARY)
```

Comment:
> It seems weird to me to do things with kubectl

This seems like a pretty reasonable punt: the alternative with `kube-rs` would be to
- Deserialize the cert-manager yaml into rust objects that can be consumed by `kube-rs`
- Create our own client via `kube-rs`
- Apply those objects to the cluster via our `kube-rs` client
- Wait / validate objects via `kube-rs` client

(essentially all the things kubectl does for us with just a static yaml manifest)

But I guess these integration tests sort of sit somewhere between a "script" and an actual program you'd compile/run. The script part of it assumes there's alot of stuff on your local host to actually accomplish the integration tests (access to the brupop testing accounts, `kubectl`, etc.)

Maybe this will be due for a refactor in the future but for now, I agree, I think this is ok.

Reply:
I think that's a reasonable ask. I remember we have that kind conversation before and I agreed to use `kube-rs` in future. I've created a new [issue](https://github.com/bottlerocket-os/bottlerocket-update-operator/issues/326) to tack this. Thanks.